release-highlights/release-highlights-2022.04/release-highlights.md

```diff
@@ -0,0 +1,53 @@
+# PatternFly Release Highlights
+## Release 2022.04
+----------------------------------------------------------
+## New features and enhancements
+
+### [Fit code editor to height](https://www.patternfly.org/v4/components/code-editor#with-sizetofit-height-height-will-growshrink-with-content)  
+
+![code editor with fit-to-height](./img/fit-to-height.png)
+
+Enables the Monaco editor's "fit-to-height" option via a prop.
+
+### [Description list - set column width](https://www.patternfly.org/v4/components/description-list#horizontal-using-custom-term-width-modifier)
+
+![description list with custom term width](./img/description-list-term-width.png)
+
+Allows for constraining the width of a column and forcing labels to wrap. This is useful when terms in the list have very different lengths.
+
+### [Predefined sizes for the avatar](https://www.patternfly.org/v4/components/avatar)
+
+The avatar component now supports predefined sizes from sm (24px) to xl (128px).
+
+### [Calendar month enhancements](https://www.patternfly.org/v4/components/calendar-month)
+
+The `date` prop is now optional which allows for opening the calendar without setting an initial date.
+
+### [Website enhancements](https://www.patternfly.org/v4/components/about-modal)
+This release also includes some PatternFly website updates including a new responsive table of contents on component pages. The TOC will transform from a vertical sidebar to a dropdown menu as the viewport shrinks.
+
+See the [latest release notes](https://www.patternfly.org/v4/developer-resources/release-notes) for a more detailed list of changes.
+
+-----------------------------------------------------------------------------
+
+## What we’re working on...
+
+### 2022.05 (April 22)
+
+* [Brand enhancements](https://github.com/patternfly/patternfly-react/issues/5637) - update the brand component to allow images to be swapped by breakpoint. This will allow a product to support unique brand images per platform (e.g. mobile vs desktop)
```

Comment:
Add period
```suggestion
* [Brand enhancements](https://github.com/patternfly/patternfly-react/issues/5637) - update the brand component to allow images to be swapped by breakpoint. This will allow a product to support unique brand images per platform (e.g. mobile vs desktop).
```